### ISSUE TITLE
GUI is blocked when starting a live channel in Kodi Nexus

### ISSUE THREAD

**Issue by chpasha** [xbmc/xbmc#23108]
### Describe the bug

Since Kodi 20, starting a live channel from PVR (Tvheadend) became blocking which is imo not good from the user point of view, since a spinner is shown, indicating that user has to wait for operation to complete. Before Kodi 20 user could just navigate away from selected channel and click another one. It was useful when having some unstable channels that would not always start within a couple of seconds. 
As we figured out in pvr.hts [bug tracker](https://github.com/kodi-pvr/pvr.hts/issues/616), inspite of the fact that spinner is shown, user still can navigate to next channel, but depending on skin used, cursor might be not visible at all, so it is not clear, which channel would be active.

## Expected Behavior

starting a live channel should not be blocking operation, user should be able to change his/her mind and navigate away to start another one

## Actual Behavior

interface is "visually blocked", but in fact, we can still navigate to other channels, but cursor might be not visible. After first channel starts,  the spinner would not show any more if we start second or third channel after that, so the behaviour is actually the same as it was in Kodi 19, 18 and many versions before. The first appearance of the spinner is not needed and not consistent.  

### To Reproduce

1. install pvr.hts (as far as I know, the simple iptv pvr was always blocking)
2. start any live channel
3. spinner is shown and would disappear only as soon as channel is running

### Debuglog

The debuglog can be found here:
https://pastebin.com/ePsqduiF

### Screenshots 
Here are some links or screenshots to help explain the problem:

## Additional context or screenshots (if appropriate)
Here is some additional context or explanation that might help:

### Your Environment
Used Operating system:

 - [ ] Android
 - [ ] iOS
 - [ ] tvOS
 - [x] Linux
 - [ ] macOS
 - [ ] Windows
 - [ ] Windows UWP

 - Operating system version/name: Coreelect 20.1-ne
 - Kodi version: 20.1.0

*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

**Comment by a1rwulf** [comment]
> interface is "visually blocked", but in fact, we can still navigate to other channels, but cursor might be not visible.

What cursor are you talking about?

Maybe it was me who added showing the spinner on first channel.
Your usecase is one of multiple possible scenarios.
Imagine using Kore or any other remote control app, you can start a channel from anywhere in the UI and have zero feedback if the start of a channel was triggered at all - hence the spinner.

**Comment by chpasha** [comment]
Hi Wolfgang,
thanks for answering

>     What cursor are you talking about?

I'm talking about the coloured rectangle, indicating which entry is currently selected in any list in kodi :-) - it is visible in non-blocking mode and not visible in blocking (but it is still there, can be moved and other channels can be started, while spinner is spinning). Or at least it is not visible in Aeon Skin, can 

>Imagine using Kore or any other remote control app

I see, but there is a visual feedback, at least in default skin - there is a play symbol and channel name appearing in top right corner. And honestly, I think that the waste majority of the users is still watching TV old school, with remote on the sofa :-) . I personally always found the non-blocking behaviour in pvr.hts great (compared to simple iptv), especially if you have multiple (sometimes unstable) sources of the same channel - you have to click a couple sometimes and definitely don't want to wait for timeout on the very first.

**Comment by ksooo** [comment]
I'm "guilty" (https://github.com/xbmc/xbmc/pull/21822) for that change, but open to revert, as I think @chpasha has a point.

**Comment by a1rwulf** [comment]
> I'm "guilty" (#21822) for that change, but open to revert, as I think @chpasha has a point.

Yeah I was involved with that and have this change in my fork too, because I don't have `there is a play symbol and channel name appearing in top right corner. ` in my skin and it's super annyoing to not see when a channel is started using a remote control app.
I understand it's debateable - not sure what the default timeout is, but I reduced the timeout to a more feasible value (somewhere around 6-8s), which works great for my usecase.
Also I usually try to stay away from unreliable source streams...

I'll leave it up to you @ksooo ;-)
I'm not a daily PVR user.
